scripts/page_browse.js

```diff
@@ -4,7 +4,7 @@
 function makePageControl(pages, selectedImageFileName, changePage) {
     // changePage is a callback to act when page changes
     const pageSelector = document.createElement("select");
-    const controls = $("<span>", {class: "nowrap control"}).append(proofIntData.strings.page + " ", pageSelector);
```

Comment:
we don't need to remove or adjust css any, right?

Reply:
It doesn't looks like we don't style anything with this class, just use it for DOM manipulation, so no CSS changes needed.